modules/@apostrophecms/piece-type/index.js

```diff
@@ -614,7 +614,7 @@ module.exports = {
 
             // Return the operation group with the new operation added.
             return {
-              name: groupName,
+              action: groupName,
```

Comment:
line 597, `action` is used as the key, not `name`.

therefore, use `action` so that the code on the line 620 works.